recipe/linter_hints/hints.toml

```
@@ -97,3 +97,7 @@ packages and bundles them all together."""
 xorg_xproto = """The use of `xorg-xextproto` and `xorg-xproto` are deprecated. \
 Please use `xorg-xorgproto` instead. It contains the latest version of all the proto \
 packages and bundles them all together."""
+xz = """The use of `xz` is deprecated. \
+If you want to link to it, use the new `liblzma-devel` package. \
+For the command line utilities, you can use `xz-tools` for `xz` and `unxz` or \
+`xz-gpl-tools` for the xz-supporting CLIs that are forks of `cat` and `grep`."""
```

Would suggest rewording this a bit to make it a bit easier on readers to see the `if...then` logic here

Also it looks like [`cat` is in `xz-tools`]( https://github.com/conda-forge/xz-feedstock/blob/ac52d61fface9ce6cd6f9c5c410ca22ff64621b1/recipe/meta.yaml#L103 ) so dropped it from the GNU tools line

```suggestion
If your package links to `liblzma`, use the new `liblzma-devel` package. \
For XZ's command line utilities (like `xz` and `unxz`), use the `xz-tools`package. \
For the xz-enabled GNU tools (like `grep`), use `xz-gpl-tools`. \
Note: Command line utilities should go in `requirements/build`."""
```